tests/Makefile

```diff
@@ -4,6 +4,7 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 DISTRO = $(shell ./os_detect)
 
 SUBDIRS := \
+	exclude_filter \
```

Comment:
Let's call it "filter_exclude", that seems to fit the naming convention a bit better.